zerver/openapi/zulip.yaml

```diff
@@ -19836,7 +19836,13 @@ components:
         id:
           type: integer
           description: |
-            The unique ID of the invitation.
+            The ID of the invitation.
+
+            Note that email invitations and reusable invitation links are stored
+            in different database tables on the server, so each ID is guaranteed
+            to be unique in combination with the boolean value of `is_multiuse`,
+            e.g. there can only be one invitation with `id: 1` and `is_multiuse:
+            true`.
```

Comment:
I originally went to update the example success response, but realized that it's totally possible for the `id` field to be the same for two different invitations, so I thought noting that here would be useful.

Reply:
Yeah, that's helpful.